app/src/main/java/invoice_automation/module/QuickBooksModule.java

```diff
@@ -83,8 +84,34 @@ public List<Customer> getAllCustomers() {
      * @return A copy of the Customer updated or created
      */
     public Customer updateCustomerFromSchool(@NonNull School school) {
-        // TODO: IA-7
-        return null;
+        boolean addNew = true;
+        // Get customer associated with the school, if it exists
+        Customer schoolCustomer = new Customer();
+        List<Customer> customers = getAllCustomers();
+        for (Customer c: customers) {
+            if (c.getDisplayName().equals(school.getSchoolName())) {
+                schoolCustomer = c;
```

Comment:
instead of storing schoolCustomer can we just set its id and sync token?

Reply:
Then instead of having an addNew flag, we can just check that the id is not null or something like that